docs/src/main/paradox/grpc-replicated-event-sourcing-transport.md

```diff
@@ -0,0 +1,99 @@
+# Akka Replicated Event Sourcing over gRPC
+
+Akka Replicated Event Sourcing extends Akka Persistence allowing multiple replicas of the same entity, all accepting
+writes, for example in different data centers or cloud provider regions. This makes it possible to implement patterns 
+such as active-active and hot standby. 
+
+Originally Akka Replicated Event Sourcing has required cross replica access to the underlying replica database which
+can be hard to open up for security and infra structure reasons.
+
+Akka Replicated Event Sourcing over gRPC builds on @ref:[Akka Projection gRPC](grpc.md) and @extref:[Akka gRPC](akka-grpc:index.html)  to instead use gRPC as the cross replica transport for events.
+
+@@@ warning
+
+This module is currently marked as [May Change](https://doc.akka.io/docs/akka/current/common/may-change.html)
+in the sense that the API might be changed based on feedback from initial usage.
+However, the module is ready for usage in production and we will not break serialization format of
+messages or stored data.
+
+@@@
+
+## Overview
+
+For a basic overview of Replicated Event Sourcing see the @extref:[Akka Replicated Event Sourcing docs](akka:typed/replicated-eventsourcing.html)
+
+Akka Replicated Event Sourcing over gRPC consists of the following three parts:
+
+The Replicated Event Sourced Behavior is run in each replica as a sharded entity using @extref:[Akka Cluster Sharding](akka:typed/cluster-sharding.html).
+
+The events of the replica is published to the other replicas using @ref:[Akka Projection gRPC](grpc.md) endpoints.
+
+Each replica consumes a number of parallel slices of the events from each other replica by running Akka Projection gRPC
+in @extref:[Akka Sharded Daemon Process](akka:typed/cluster-sharded-daemon-process.html).
+
+
+
+## Dependencies
+
+To use the R2DBC module of Akka Projections add the following dependency in your project:
+
+@@dependency [sbt,Maven,Gradle] {
+group=com.lightbend.akka
+artifact=akka-projection-grpc_$scala.binary.version$
+version=$project.version$
+}
+
+Akka Replicated Event Sourcing over gRPC require Akka 2.8.0 or later and can only be run in an Akka cluster since it uses cluster components.
+
+The functionality is provided through the `akka-projection-grpc` module. 
+
+@@project-info{ projectId="akka-projection-grpc" }
+
+### Transitive dependencies
+
+The table below shows `akka-projection-grpc`'s direct dependencies, and the second tab shows all libraries it depends on transitively.
+
+@@dependencies{ projectId="akka-projection-grpc" }
+
+
+### Serialization of events
+
+FIXME is there anything here?
```

Comment:
I think I've arrived at the same conclusion as well, but I'll mention that it will use the same serialization that was set up for serializing the events. And maybe mention something about versioning, since the serialization format now has gone from an internal detail to a inter-service concern (even if it likely just is different clusters/deployments of the same service)